src/js/app.js

```diff
@@ -0,0 +1,5 @@
+global.onIndexLoad = require('./index').onIndexLoad;
```

Comment:
I thought this file got generated?

Reply:
It does-- I just unintentionally chose the same name for the entry point (`src/js/app.js`) and the final result (`dist/js/app.js` and `dist/js/app.min-*.js`).  Either could be changed to something else (e.g. `src/js/entry.js`) to make things unique.

Reply:
To avoid confusion I think that would be wise.

Reply:
Okay, I've renamed it to `entry.js` (924b2409e12162808d2e010ba5fe6241eb7d9264).